src/components/Attributes.js

```diff
@@ -7,30 +7,37 @@ import { Card, HeaderBar } from './CustomStyled';
 import { calculateModifier } from '../services/helper';
 
 
-function Attributes({ character }) {
+function Attributes({ character, update, disabled }) {
+  function onChange(field) {
+    return (e) => {
+      const val = +e.target.value;
```

Comment:
What's this notation doing? I've never seen `= +`

Reply:
if you put a `+` in front of a string it converts it to a number if possible

Reply:
basically short for parseInt `(e.target.value)`